lib/response-transforms.js

```diff
@@ -32,4 +32,8 @@ module.exports = {
       });
     }
   },
+  jsonify: res => {
```

Comment:
Accept header is irrelevant. The server isn't setting content-type, so when it gets to us here it's considered plain text. I'd originally fixed the server, but eventually settled on testing the behavior that IS instead of the behavior that SHOULD be.